binderhub/tests/test_auth.py

```diff
@@ -94,6 +101,22 @@ def reset():
         # allow container of statuses
         assert r.status_code in status
 
+    ban_message = "Requests from local are not allowed"
     if status == 403:
         # check error message on 403
-        assert "Requests from local are not allowed" in r.text
+        assert ban_message in r.text
+
+    if banned and path.startswith("/build"):
+        # /build/ is event-stream, so allow connecting with200
```

Comment:
```suggestion
        # /build/ is event-stream, so allow connecting with status 200
```
i think that is the missing word here